graph/pull.go

```diff
@@ -491,7 +491,7 @@ func (s *TagStore) pullV2Tag(eng *engine.Engine, r *registry.Session, out io.Wri
 
 				io.Copy(tmpFile, utils.ProgressReader(ioutil.NopCloser(tarSumReader), int(l), out, sf, false, utils.TruncateID(img.ID), "Downloading"))
 
-				out.Write(sf.FormatProgress(utils.TruncateID(img.ID), "Verifying Checksum", nil))
+				out.Write(sf.FormatProgress(utils.TruncateID(img.ID), "Verifying Checksum - this is a tech preview, don't rely on it for security yet.", nil))
```

Comment:
I think it does need the preview subtext and I believe @shykes thought so
as well unless I misunderstood.

On Wed, Jan 21, 2015 at 3:13 PM, Derek McGowan <notifications@github.com>
wrote:

> In graph/pull.go
> <https://github.com/docker/docker/pull/10235#discussion_r23343298>:
>
> > @@ -491,7 +491,7 @@ func (s *TagStore) pullV2Tag(eng *engine.Engine, r *registry.Session, out io.Wri
> >
> >  				io.Copy(tmpFile, utils.ProgressReader(ioutil.NopCloser(tarSumReader), int(l), out, sf, false, utils.TruncateID(img.ID), "Downloading"))
> >
> > -				out.Write(sf.FormatProgress(utils.TruncateID(img.ID), "Verifying Checksum", nil))
> > +				out.Write(sf.FormatProgress(utils.TruncateID(img.ID), "Verifying Checksum - this is a tech preview, don't rely on it for security yet.", nil))
>
> I think the only other place it belongs is next to the verified message.
> Keep in mind the current verified message will only show up for official
> images signed by official image build key. While we might change the
> checksum method, the fact that we are verifying the checksum using tarsum
> IMO does not need to need the preview subtext.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/10235/files#r23343298>.
>